testdata/p4_16_samples/arith-skeleton-d.p4

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
While porting, we should ignore the tests which does not involve any backend processing. This will not compile with ./p4c-dpdk I assume and our regression script will not run this, does it?